altair/utils/save.py

```diff
@@ -111,6 +122,7 @@ def save(
             vegalite_version=vegalite_version,
             vegaembed_version=vegaembed_version,
             embed_options=embed_options,
+            template="inline" if inline else "standard",
```

Comment:
Would this prevent the use of `'universal'` as a value for the `template` parameter? [It seems like that is used in a the `display.py`](https://github.com/altair-viz/altair/search?q=universal) files in both the `vega` and `vega-lite` folders.

Reply:
Yeah, I suppose it would. But I don't think `universal` would work anyway because it doesn't write a full HTML document (only an outer `<div>` element).

But your point still stands that this would prevent the use of a custom Jinja template object. I'll update it to only overwrite the template if `inline=True`

Reply:
Done in 90baae7b2876232bd9e9224249a685dc5fcfe9a6